include/cross.h

```diff
@@ -161,4 +161,13 @@ bool get_expanded_files(const std::string &path,
 
 std::string get_language_from_os();
 
+#ifndef WIN32
+#define XATTR_FATTR_NAME "user.dosbox_fattr"
+#define XATTR_FATTR_LEN 9 // 8 bits + null terminator
+
+uint16_t get_fat_attribs_from_xattr(const char *path);
+int set_xattr_from_fat_attribs(const char* path, uint16_t fattr);
```

Comment:
Let's use `dos_attrib`; that's the least ambiguous name (better than `fattr` which is very cryptic or `fat_attrib` even).

Make args const whenever possible:

`const uint16_r dos_attrib`

Reply:
I'd suggest these name:

` get_dos_attribs_from_extended_attrib`
` set_extended_attrib_from_dos_attribs`

But at the very least rename `fat_attrib` to `dos_attrib`.